### PR TITLE
Document LTS versions supporting ACME v2

### PR DESCRIPTION
**Pull request by thedewi**
I thought it might be a good idea to document the LTS versions for this fix. Is this something we do? Is there a more readable way to word it?

### CODE REVIEW

File: docs/administration/security/exposing-octopus/lets-encrypt-integration.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ position: 2300
 
 **Octopus 3.16** or newer can integrate with [Let's Encrypt](https://g.octopushq.com/LetsEncryptOrg) to setup and manage the SSL certificate for the Octopus Portal. When the certificate nears its expiration date, Octopus will automatically renew the certificate with no intervention required.
 
-**Octopus 2019.10.3** or newer uses ACME v2, which is required after Let's Encrypt [retired the v1 APIs](https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430) in November 2019.
+**Octopus 2019.10.3, 2019.9.6, 2019.6.12** or newer use ACME v2, which is required after Let's Encrypt [retired the v1 APIs](https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430) in November 2019.
```
Comment (wordlee) [inline docs/administration/security/exposing-octopus/lets-encrypt-integration.md]: Perhaps call out that these are LTS versions:
```suggestion
Long-term support versions: **Octopus 2019.10.3, 2019.9.6, 2019.6.12** or newer use ACME v2, which is required after Let's Encrypt [retired the v1 APIs](https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430) in November 2019.
```

File: docs/administration/security/exposing-octopus/lets-encrypt-integration.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ At this point, we recommend enabling [Force SSL](/docs/administration/security/e
 
 ### ACME v1 retirement
 
-After Let's Encrypt [retired the v1 APIs](https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430) in November 2019, **Octopus 2019.10.3** or newer is required to use this feature.
+After Let's Encrypt [retired the v1 APIs](https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430) in November 2019, **Octopus 2019.10.3, 2019.9.6, 2019.6.12** or newer are required to use this feature.
```
Comment (wordlee) [inline docs/administration/security/exposing-octopus/lets-encrypt-integration.md]: And again here: 
```suggestion
After Let's Encrypt [retired the v1 APIs](https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430) in November 2019, LTS versions: **Octopus 2019.10.3, 2019.9.6, 2019.6.12** or newer are required to use this feature.
```

**Comment by thedewi** [comment]
Oh, just the last two are LTS. The download page just puts "LTS" after version numbers, so I've done that now. What do you think?

**Comment by thedewi** [comment]
Thanks!

**Pull request merged**